.storybook-image-snapshots/config.js

```diff
@@ -0,0 +1,7 @@
+import { configure } from '@storybook/react';
+
+function loadStories() {
+	require('../components/storybook-stories-image-snapshots'); // eslint-disable-line global-require
```

Comment:
It's not obvious to me why you are doing this this way. Especially since you have to disable an eslint rule...

Reply:
It's a CJS way of bringing in the variables. I'm open to another way. It was not a problem I was trying to solve.

The library is ES6, except when it's not--which is mostly in Node which is CJS. Until node supports imports. I figure we'll need it like this and so we have a eslint comment